code/src/components/StartPage.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import "antd/dist/antd.css";
+
+import labyrinth, { generateGame } from '../reducers/labyrinth';
+
+import { CardContainer, Title, TitleBar, Content, SecondaryText, Button} from './Styles';
+
+const StartPage  = () => {
+    const [player, setPlayer] = useState('');
+    const [notValidated, setNotValidated] = useState(false);
+
+    const dispatch = useDispatch();
+
+    const onPlayerSet = () => {
+        if (player) {
+            dispatch(labyrinth.actions.setPlayer(player));
+            dispatch(generateGame());
+        } else {
+            setNotValidated(true)
+        }
+        
+    }
+
+    return (
+        <CardContainer>
+            <Content>
+                <TitleBar>
+                    <Title>Welcome to Labyrinth! Please enter your name: </Title>
+                    {notValidated && <SecondaryText>Please enter your name</SecondaryText>}

```

Comment:
If you set a minimum number of characters, then the SecondaryText could be a bit more descriptive as well 😄